script/changelog/parser.ts

```diff
@@ -55,9 +55,34 @@ export function findIssueRef(body: string): string {
   return issueRef
 }
 
-function getChangelogEntry(commit: IParsedCommit, pr: IAPIPR): string {
+type ReleaseNotes =
+  | {
+      readonly kind: 'found' | 'default'
+      readonly text: string
+    }
+  | {
+      readonly kind: 'omitted'
+    }
+
+export function getReleaseNotesDescription(pr: IAPIPR): ReleaseNotes {
+  const re = /[Nn]otes: (.*)/gm
```

Comment:
Should this regex start with a `^`? That would stop it from matching `notes:` in the middle of a sentence.

Reply:
👍 I mostly opened this early to get feedback but I'll throw some better test cases at it while we talk about the goals here.